File: app/src/main/java/com/sagar/watchnext/network/models/movies/MovieList.java

package com.sagar.watchnext.network.models.movies;

import android.util.Log;

import java.util.List;

import com.google.gson.annotations.SerializedName;

public class MovieList {

    @SerializedName("page")
    private int page;

    @SerializedName("results")
    private List<Movie> movies;

    @SerializedName("dates")
    private Dates dates;

    @SerializedName("total_pages")
    private int totalPages;

    @SerializedName("total_results")
    private int totalResults;

    public int getPage() {
        return page;
    }

    public List<Movie> getMovies() {
        return movies;
    }

    public Dates getDates() {
        return dates;
    }

    public int getTotalPages() {
        return totalPages;
    }

    public int getTotalResults() {
        return totalResults;
    }
}

File: app/src/main/java/com/sagar/watchnext/network/models/movies/MovieList.kt
package com.sagar.watchnext.network.models.movies

import com.google.gson.annotations.SerializedName

class MovieList {

    @SerializedName("page")
    val page: Int = 0

    @SerializedName("results")
    val movies: List<Movie>? = null

    @SerializedName("dates")
    val dates: Dates? = null

    @SerializedName("total_pages")
    val totalPages: Int = 0

    @SerializedName("total_results")
    val totalResults: Int = 0
}
